home_assistant_datasets/tool/assist/collect.py

```diff
@@ -144,6 +174,7 @@ def run(args: argparse.Namespace) -> int:
         f"--dataset={args.dataset or ''}",
         f"--model_output_dir={args.model_output_dir or ''}",
         f"--categories={args.categories or ''}",
+        f"--count={args.count or ''}",
     ]
     if args.test_path:
```

Comment:
Given this is an integer flag I was expecting to not see a string value here. Does this work?

```suggestion
    ],
    if args.count:
        pytest_args.append(f"--count={args.count}")
```

Reply:
Yep that works!